kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/materials.js

```diff
@@ -1241,7 +1241,7 @@ onEvent('server.datapack.low_priority', (event) => {
                 {
                     type: 'tconstruct:material_fluid',
                     fluid: {
-                        name: 'emendatusenigmatica:molten_enderium',
+                        name: 'tconstruct:molten_enderium',
```

Comment:
If you use a tag here instead of a fluid, it would work with both Tinkers and Emendatus Enigmatica fluids. Most of the native casting/alloy recipes use the forge tag.